app/assets/stylesheets/views/_homepage.scss

```diff
@@ -174,6 +111,16 @@
   }
 }
 
+// Overrides
+
+// Ensures the card text takes up the full width of the container
+.gem-c-cards__list-item-wrapper {
```

Comment:
Could this be done as an option on the component itself?

Reply:
Yes definitely, was thinking the same thing